### PR TITLE
0.2.0.1

### PR DESCRIPTION
**Pull request by Limezy**

### CODE REVIEW

**Comment by Limezy** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -1,103 +0,0 @@
-packaging_format = 2
```
Comment (alexAubin) [inline manifest.toml:1]: (NB: the point is that we're ideally supposed to use packaging v2 now, not the old v1 .. though it still works :grimacing: )

Comment (Limezy) [reply manifest.toml:1]: Yes, I have this in mind...
But it will take a bit of time for me to dig in and learn the new format 😅

Comment (Limezy) [reply manifest.toml:1]: Also, I don't know anything about Nim, so I took the path of copy pasting Nitter package (v1) and adapt it for Buckets (and I have to say that it worked surprisingly well. Never got a nearly working package so fast)

**Pull request merged**